autogen/main.tf.tmpl

```diff
@@ -91,6 +91,22 @@ resource "google_compute_region_instance_group_manager" "{{ module_name }}" {
     }
   }
 
+  dynamic "stateful_internal_ip" {
```

Comment:
I believe we can condense this to one dynamic block if we use a single variable with a external/internal flag on the object.